src/client/options/mod.rs

```diff
@@ -2115,6 +2121,17 @@ impl ConnectionString {
                     ))
                 }
             },
+            "uuidrepresentation" => {
+                match value.to_lowercase().as_str() {
+                    "csharplegacy" => self.uuid_representation = Some(UuidRepresentation::CSharpLegacy),
+                    "javalegacy" => self.uuid_representation = Some(UuidRepresentation::JavaLegacy),
+                    "pythonlegacy" => self.uuid_representation = Some(UuidRepresentation::PythonLegacy),
+                    _ => return Err(ErrorKind::InvalidArgument { 
+                        message: format!("connection string `uuidRepresentation` option can be one of `csharpLegacy`, `javaLegacy`, or `pythonLegacy`. Received invalid `{value}`") 
```

Comment:
Sadly, our MSRV doesn't support inline format identifiers, so this will need to use the older format.